gateway/src/app.ts

```diff
@@ -42,6 +43,37 @@ app.get('/', (_req: Request, res: Response) => {
   res.status(200).json({ status: 'ok' });
 });
 
+app.get('/status', async (_req: Request, res: Response) => {
+  const avalanche = AvalancheRoutes.avalanche;
+  const ethereum = EthereumRoutes.ethereum;
+  const connectedNetworks = [];
+  try {
+    const avalancheNetwork = await getConnectionInformation(avalanche);
+    connectedNetworks.push(avalancheNetwork);
+  } catch (err) {
+    logger.error(err);
+  }
+  try {
+    const ethNetwork = await getConnectionInformation(ethereum);
+    connectedNetworks.push(ethNetwork);
+  } catch (err) {
+    logger.error(err);
+  }
+
+  res.status(200).json({
+    connectedNetworks,
+  });
+});
+
+async function getConnectionInformation(connector: EthereumBase) {
+  return {
+    chainName: connector.chainName,
+    chainId: connector.chainId,
+    rpcUrl: connector.rpcUrl,
+    currentBlockNumber: await connector.getCurrentBlockNumber(),
```

Comment:
Also, I think `connected` status would be great here. `currentBlockNumber` can be refactored to have a negative value if we're not connected to the network.

Reply:
do you mean returning just a string? Like 'connected'

Reply:
I mean something like `{ connected: true | false }`